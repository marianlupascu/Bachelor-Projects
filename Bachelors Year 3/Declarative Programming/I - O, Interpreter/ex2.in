Ion Ionel Ionescu, 70
Gica Petrescu, 99
Mustafa ben Muhamad, 7
Ionica, 99