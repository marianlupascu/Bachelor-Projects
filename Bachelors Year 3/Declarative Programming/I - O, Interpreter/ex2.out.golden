Cel mai in varsta este Gica Petrescu (99 ani).
